templates/user/my-courses.php

```diff
@@ -15,8 +15,7 @@
 <?php
 /**
  * Executes before the Sensei my courses markup begins. This hook
- * only fires whe a user is logged in. If you need to add
- * something here for when users are logged out use `sensei_login_form_before`.
```

Comment:
Yes. I'd say this change in general is a breaking change in those cases, so we probably want to document it as such.